packages/design-system/src/components/N8nNodeCreatorNode/NodeCreatorNode.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<div
+		:class="{
+			[$style.creatorNode]: true,
+			[$style.padingless]: padingless,
+			[$style.hasAction]: !showActionArrow,
+		}"
+		v-on="$listeners"
+		v-bind="$attrs"
+	>
+		<div :class="$style.nodeIcon">
+			<slot name="icon" />
+		</div>
+		<div>
+			<div :class="$style.details">
+				<span :class="$style.name" v-text="title" data-test-id="node-creator-item-name" />
+				<trigger-icon v-if="isTrigger" :class="$style.triggerIcon" />
+				<n8n-tooltip v-if="!!$slots.tooltip" placement="top">
+					<template #content>
+						<slot name="tooltip" />
+					</template>
+					<n8n-icon :class="$style.tooltipIcon" icon="cube" />
+				</n8n-tooltip>
+			</div>
+			<p :class="$style.description" v-if="description" v-text="description" />
+		</div>
+		<slot name="dragContent" />
+		<button :class="$style.panelIcon" v-if="showActionArrow">
+			<font-awesome-icon :class="$style.panelArrow" icon="arrow-right" />
+		</button>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import TriggerIcon from './TriggerIcon.vue';
+import N8nTooltip from '../N8nTooltip';
+
+export interface Props {
+	active?: boolean;
+	isTrigger?: boolean;
+	description?: string;
+	title: string;
+	padingless?: boolean;
```

Comment:
That's a good point. I should have used a CSS variable with a default which could then be overwritten from a parent. But since this prop is not even used anymore, I've removed it.